src/main/resources/validationrules/prm/prm-role-mapping-validation.drl

```diff
@@ -0,0 +1,39 @@
+package validationrules.core;
+import uk.gov.hmcts.reform.roleassignment.domain.model.RoleAssignment;
+import uk.gov.hmcts.reform.roleassignment.domain.model.Request;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.Status;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RequestType;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleCategory;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleType;
+import function uk.gov.hmcts.reform.roleassignment.domain.service.common.ValidationModelService.logMsg;
+
+rule "prm_create_org_role"
+when
+    $rq: Request(
+             clientId == "am_org_role_mapping_service",
+             process == "professional-organisational-role-mapping",
+             reference != null)
+    $ra: RoleAssignment(
+             status == Status.CREATE_REQUESTED,
+             roleType == RoleType.ORGANISATION,
+             roleCategory == RoleCategory.PROFESSIONAL)
+then
+    $ra.setStatus(Status.CREATE_APPROVED);
+    $ra.log("Create approved : prm_create_org_role");
+    update($ra);
+    logMsg("Rule : prm_create_org_role");
+end;
+
+rule "prm_delete_org_role"
```

Comment:
Please add comment before rule similar to other ORM comment https://github.com/hmcts/am-role-assignment-service/blob/88dcba9706fd3467b5b91ea6bcaec724fd933441/src/main/resources/validationrules/core/organisational-role-mapping-common.drl#L34-L38

Reply:
added comment about delete rule on commit [f8269a9](https://github.com/hmcts/am-role-assignment-service/pull/2062/commits/f8269a94896e32c2d336b6c76aa7ff585d45c1b5)